### ISSUE TITLE
RPA.Windows - Ability to check toggle state of Checkbox

### ISSUE THREAD

**Issue by savitakul** [robocorp/rpaframework#1166]
We are using RPA.Windows for desktop automation, we are trying to find ways to get the toggle state of checkbox and act on UI accordingly.
We do not find any RPA.Windows keyword that provides this information.
Can you please let us know if we can have such keyword for checkbox control?

**Comment by mikahanninen** [comment]
Yes, that can be done. I will add related functionality to the library.

**Comment by KamilSoko** [comment]
Hi,

you can always define such keyword in custom library like this:
1. custom_lib.py
`@keyword
def get_checkbox_state_py(checkbox_control: CheckBoxControl) -> bool:
    toggle = checkbox_control.GetTogglePattern()
    return toggle.ToggleState`
2. custom_resource.resource
`Get Checkbox State
    [Arguments]    ${checkbox_locator}

    ${checkbox_control} =    Get Element    ${checkbox_locator}
    ${state} =    Get Checkbox State Py    ${checkbox_control.item}
    RETURN    ${state}`

**Comment by MoonOnHead** [comment]
I am also looking for this functionality, can you please update once its implemented.
@KamilSoko can you please give more details like library references n all

**Comment by kmyllykoski** [comment]
Reading toggle state of a Checkbox in Windows applications using RPA.Windows would be nice.
